src/azul/queues.py

```diff
@@ -148,6 +148,20 @@ def _is_azul_queue(self, queue_name) -> bool:
         return any(config.unqualified_resource_name_or_none(queue_name, suffix)[1] == config.deployment_stage
                    for suffix in ('', '.fifo'))
 
+    @staticmethod
```

Comment:
```suggestion
    @classmethod
```
@noah-aviel-dove is adding a clause explaining this:

https://github.com/DataBiosphere/azul/pull/1428/files#diff-ea1f65a8eaac0cec7ae151dfdbbbc587R352